webpack.config.babel.js

```diff
@@ -135,10 +135,10 @@ const basePlugins = [
     'DESKTOP_SYNC_URL': CONFIG.DESKTOP_SYNC_URL,
     'MOBILE_URL' : CONFIG.MOBILE_URL,
     'SMS_DELIVERY_URL' : CONFIG.SMS_DELIVERY_URL,
-    // Increment BASE_36_VERSION with each release following Base32 notation, i.e AA -> AB
+    // Increment BASE_32_VERSION with each release following Base32 notation, i.e AA -> AB
     // Do it only when we introduce a breaking change between SDK and cross device client
     // ref: https://en.wikipedia.org/wiki/Base32
-    'BASE_36_VERSION' : 'AA',
+    'BASE_32_VERSION' : 'AB',
```

Comment:
Do we need to bump this version? Has there been a breaking change?

Reply:
The SMS feature is not a breaking change but it's a major change of the cross device feature so I thought we might want to keep track of that.

Reply:
According to guidelines, we should `Increment BASE_32_VERSION with each release...`. @hjheath do you challenge that? I think bumping version each time may mitigate the risk of forgetting to deploy when real changes occur, especially when many different people are starting to do the releases

Reply:
yeah I think it makes sense to bump it for this release. for patch releases we should not need to.

Reply:
so this is an interesting one. if there are no breaking changes, we can not bump it, but we will always redeploy it right?

And coming back to the version bumping, a UI refresh might not have a major bump, but if we don't distinguish the two, either the new version doesn't get the UI refresh or the old one gets it and it looks inconsistent.